docs/development.rst

```diff
@@ -213,6 +230,103 @@ active, as long as the Python `tox` package is installed in it:
     $ tox -e py27                      # Run tests on Python 2.7
 
 
+.. _`Testing from the source archives on Pypi or GitHub`:
+
+Testing from the source archives on Pypi or GitHub
+--------------------------------------------------
+
+The wheel distribution archives on Pypi
+(e.g. ``pywbem-1.0.0-py2.py3-none-any.whl``)
+contain only the files needed to run pywbem, but not the files needed to test
+it.
+
+The source distribution archives on Pypi and GitHub
+(e.g. ``pywbem-1.0.0.tar.gz``)
+contain all files that are needed to test pywbem.
+
+When installing these source distribution archives, only the files needed for
+running pywbem are installed into the active Python environment, but not the
+test files.
+
+The following commands download the source distribution archive on Pypi for a
+particular version of pywbem into the current directory and unpack it:
+
+.. code-block:: bash
+
+    $ pip download --no-deps --no-binary :all: pywbem==1.0.0
+    $ tar -xf pywbem-1.0.0.tar.gz
+    $ cd pywbem-1.0.0
+    $ ls -1
+    -rw-r--r--   1 maiera  staff    468 Jun 29 22:31 INSTALL.md
+    -rw-r--r--   1 maiera  staff  26436 May 26 06:45 LICENSE.txt
+    -rw-r--r--   1 maiera  staff    367 Jul  3 07:54 MANIFEST.in
+    -rw-r--r--   1 maiera  staff   3451 Jul  3 07:55 PKG-INFO
+    -rw-r--r--   1 maiera  staff   7665 Jul  2 23:20 README.rst
+    -rw-r--r--   1 maiera  staff   1624 Jul  2 23:20 README_PYPI.rst
+    -rwxr-xr-x   1 maiera  staff   2881 Jun 29 22:31 build_moftab.py
+    -rwxr-xr-x   1 maiera  staff  13850 Jun 29 22:31 mof_compiler
+    -rw-r--r--   1 maiera  staff    105 May 26 06:45 mof_compiler.bat
+    drwxr-xr-x  29 maiera  staff    928 Jul  3 07:55 pywbem
+    drwxr-xr-x   8 maiera  staff    256 Jul  3 07:55 pywbem.egg-info
+    drwxr-xr-x  18 maiera  staff    576 Jul  3 07:55 pywbem_mock
+    -rw-r--r--   1 maiera  staff   1067 Jun 29 22:31 requirements.txt
+    -rw-r--r--   1 maiera  staff     38 Jul  3 07:55 setup.cfg
+    -rwxr-xr-x   1 maiera  staff   7555 Jul  3 07:24 setup.py
+    -rw-r--r--   1 maiera  staff   2337 Jul  2 23:20 test-requirements.txt
+    drwxr-xr-x  15 maiera  staff    480 Jul  3 07:55 tests
+
+Pywbem, its dependent packages, and packages needed for testing pywbem can be
```

Comment:
Maybe I am missing it, but isn't the issue to provide for executing the pywbem tests as part of the installation whereas the PYPI installation does not do that.  I could not find that goal statement  itself in the documentation.

Reply:
Added the following a few lines up, as a goal statement and motivation:
```
This allows testing pywbem without having to check out the entire repository,
and is convenient for testing e.g. when packaging pywbem into OS-level packages.
```